presto-main/src/test/java/com/facebook/presto/execution/TestSqlTaskExecution.java

```diff
@@ -168,7 +168,9 @@ public void testSimple(PipelineExecutionStrategy executionStrategy)
                             OptionalInt.empty(),
                             executionStrategy)),
                     ImmutableList.of(TABLE_SCAN_NODE_ID),
-                    executionStrategy == GROUPED_EXECUTION ? StageExecutionDescriptor.fixedLifespanScheduleGroupedExecution(ImmutableList.of(TABLE_SCAN_NODE_ID)) : StageExecutionDescriptor.ungroupedExecution());
+                    executionStrategy == GROUPED_EXECUTION
+                            ? StageExecutionDescriptor.fixedLifespanScheduleGroupedExecution(ImmutableList.of(TABLE_SCAN_NODE_ID), 8)
```

Comment:
nit: could we use a constant here (if there's one?)

Reply:
@arhimondr : Because the largest driver group for this pipeline is 7: https://github.com/prestodb/presto/blob/f65d4d9d840e06f742659f7ecf4b6cd95d342403/presto-main/src/test/java/com/facebook/presto/execution/TestSqlTaskExecution.java#L283

Alliteratively, we can just use a number (e.g. 16). Since it's test code and it doesn't really check the number of total lifespans. What do you think?

Reply:
We can leave it as is. I just though it is deduced from some other constant around.